src/App.js

```diff
@@ -31,7 +31,7 @@ class App extends Component {
               user={this.props.data.user}
             />}
         />
-        <Route path="/user/edit" component={UserInfo} />
+        <Route path="/user/edit" component={MemberProfileEdit} />
```

Comment:
It seems like for routes if we are using `/user/:id` for seeing users generally, then maybe we want to have something different for this route as the logged in user. Maybe `/me/edit`, or `/profile/edit` or something?